aws_cloudtrail.tf

```diff
@@ -6,28 +6,29 @@ resource "aws_cloudtrail" "missing-multi-region" {
   name                          = "tf-trail-foobar"
   s3_bucket_name                = "some-s3-bucket"
   s3_key_prefix                 = "prefix"
-  include_global_service_events = false
+  include_global_service_events = true
 }
 
 resource "aws_cloudtrail" "false-multi-region" {
   name                          = "tf-trail-foobar"
   s3_bucket_name                = "some-s3-bucket"
   s3_key_prefix                 = "prefix"
-  include_global_service_events = false
-  is_multi_region_trail         = false 
+  include_global_service_events = true
+  is_multi_region_trail         = true
+
 }
 
 resource "aws_cloudtrail" "missing-kms" {
   name                          = "missing-kms"
   s3_bucket_name                = "some-s3-bucket"
```

Comment:
**Accurics remediation**
 
**[MEDIUM]: Enable AWS CloudTrail logging for global services**
 
**Vulnerability**
Ensure AWS CloudTrail trails track API calls for global services such as IAM, STS and CloudFront.
 
**Remediation**
Ensure that include_global_service_events is set to true

source: accurics-